cmake/EthDependencies.cmake

```diff
@@ -29,8 +48,12 @@ if (DEFINED MSVC)
 	endif()
 	set (CMAKE_PREFIX_PATH ${ETH_DEPENDENCY_INSTALL_DIR} ${CMAKE_PREFIX_PATH})
 
-	# TODO it windows SDK is NOT FOUND, throw ERROR
+	# TODO if windows SDK is NOT FOUND, throw ERROR
 	# from https://github.com/rpavlik/cmake-modules/blob/master/FindWindowsSDK.cmake
+	#
+	# TODO What are we using WindowsSDK for specifically?  Do we need to do that, or
```

Comment:
Yeah - looks redundant.   I'm trying chopping it out, and will add that to the PR if all is well.
Ditto for the "gen" include path just below, which I think is also unneeded.